infrastructure/ansible/roles/traffic_ops/tasks/traffic_ops.yml

```diff
@@ -122,6 +122,12 @@
     dest: "{{ to_tvdb_aes_key_loc }}"
   notify: Restart Traffic Ops
 
+- name: Remove LDAP config if LDAP not used
+  file:
+    path: "{{ to_conf_installdir }}/ldap.conf"
+    state: absent
+  when: to_ldap_setup == "no" 
```

Comment:
This needs to use actual boolean evaluation instead of string matching

Reply:
Done